kitsune/wiki/views.py

```diff
@@ -166,6 +166,11 @@ def document(request, document_slug, template=None, document=None):
     else:
         document_css_class = ''
 
+    if request.MOBILE and 'minimal' in request.GET:
```

Comment:
Just looking at the minimal parameter will also make it easier for us to work on BuddyUp on desktop. We won't have to change our UA ;)